arq/connections.py

```diff
@@ -90,13 +88,15 @@ def __init__(
         job_serializer: Optional[Serializer] = None,
         job_deserializer: Optional[Deserializer] = None,
         default_queue_name: str = default_queue_name,
+        default_job_expiry: int = expires_extra_ms,
```

Comment:
```suggestion
        expires_extra_ms: int = expires_extra_ms,
```
Might not be the best name, but this will be clearer as a change.